spec/frodo/services/online_orderer_finder_spec.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+describe Frodo::OnlineOrdererFinder do
+  let(:body) { JSON.unparse(user) }
+  let(:headers) do
+    { 'AUTHORIZATION' => token }
+  end
+  let(:status) { 200 }
+  let(:token) { 'Bearer test-token-here' }
+  let(:url) { "#{ENV['GANDALF_URL']}/organizations/#{org_id}/#{online_orderer}" }
+  let(:org_id) { '6940f407-06d2-480e-a52c-002ff7b28503' }
+  let(:online_orderer) { 'family' }
+  let(:user_id) { '6940f407-06d2-480e-a52c-002ff7b28503' }
+
+  before do
+    ENV['GANDALF_URL'] = 'https://www.gandalf.com'
```

Comment:
Setting the environment variable explicitly in tests isn't a good practice because `ENV` is global, which means it will leak out into other tests and can cause unintended behavior. 

A safer way to do this is to use a mock: 

```
    allow(ENV).to receive(:[]).with('GANDALF_URL').and_return('https://www.gandalf.com')
```

You can read more about `allow` here if you want: https://relishapp.com/rspec/rspec-mocks/v/2-99/docs/method-stubs/allow-with-a-simple-return-value